docs/Practices/product-analytics.md

```diff
@@ -0,0 +1,82 @@
+
+### Capturing user data
+
+We capture user data in the spirit of [The Segment Spec](https://segment.com/docs/connections/spec/) -- the linked documentation is extremely useful for anyone engaging in product analytics. We identify users with a stable, unique identifier that available across all sources: their Prenda Hub ID. We respect and advantage [the reserved user traits](https://segment.com/docs/connections/spec/identify/#traits) that Segment defines. We track the actions that users take with a unique name and custom set of properties.
+
+Our custom event properties do not include user identity information or other general metadata. Segment takes care of associating events with users and their identity information, and tracking metadata like location and time ([example event in json format](https://segment.com/docs/connections/spec/track/#example)).
+
+Our event names follow a standard format: **Location - Sub-location (optional) - Object Action**. Event properties
+
+The **location** is your application (or other unique source). The **object** (a noun) is the piece of the app that the user is interacting with. The **action** (a past tense verb) is how user interacts with that object -- it is a . A **sub-location** can be used to help distinguish events that would otherwise have the same name or that benefit from being grouped by sub-page, area, or feature -- it is optional. For each part, use Proper Case. For example,
+
+- Prenda World - Goal Updated
+- Prenda World - Microschool Profile Updated
+- Prenda World - Payment Information - Read More Link Clicked
+- Marketplace - Start Enrollment Clicked
+- Marketplace - Guide Biography Expanded
+- Marketplace - Grades Filtered
+- Marketplace - Main Search - Map Dragged
+- Marketplace - Microschool Details - Map Dragged
+
+Sometimes, the object isn't exactly a "piece of the app" but is a reasonable transformation. For example, in "Prenda World - User Logged In" could have been "Prenda World - Account Logged In" because "account" is closer to a piece of the app, however "user" is a friendlier object name for future analysis.
+
+It is _not_ acceptable to break the standard format unless _absolutely_ necessary.
+
+Our event name standard is a combination of [Segment's "naming conventions for clean data"](https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/) and [Heap's location prefixing under "naming conventions"](https://help.heap.io/data-management/data-management-features/naming-conventions-overview/). Both linked articles are great resources to review for clarity on why a standard is useful, what ours is composed from, and how it can be applied.
+
+:::caution Troubleshooting Heap
+
+The Segment to Heap integration has peculiarities you should be aware of,
+
+- Only the `identify` and `track` calls can flow into Heap.
```

Comment:
```suggestion
- Only the `identify` and `track` calls can flow into Heap. [See "Destination Info" here.](https://segment.com/docs/connections/destinations/catalog/heap/)
```

Reply:
That is linked in the last bit of the section. see my response to your comment below as for what I was hoping to go for by not linking something for every bullet point.